pkg/platform/packet/packet.go

```diff
@@ -88,8 +89,9 @@ type config struct {
 	ReservationIDs           map[string]string `hcl:"reservation_ids,optional"`
 	ReservationIDsDefault    string            `hcl:"reservation_ids_default,optional"`
 	CertsValidityPeriodHours int               `hcl:"certs_validity_period_hours,optional"`
-	KubeAPIServerExtraFlags  []string          `hcl:"kube_apiserver_extra_flags,optional"`
-	NodesDependOn            []string          // Not exposed to the user
+	Oidc                     *oidc.Config      `hcl:"oidc,block"`
+	KubeAPIServerExtraFlags  []string
+	NodesDependOn            []string // Not exposed to the user
```

Comment:
Maybe put this comment on top of all "non-user-accessible" fields?

Reply:
I'd rather have this change in the commit where we introduce `KubeAPIServerExtraFlags`, but I'm not blocking on this. Just a comment for the next time.

Reply:
Addressed the change in the first commit where `KubeAPIServerExtraFlags` is introduced.